pkgs/top-level/python-packages.nix

```diff
@@ -3173,6 +3196,21 @@ let
     };
   };
 
+  easy-process = buildPythonPackage rec {
+    name = "EasyProcess-0.1.9";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/E/EasyProcess/${name}.tar.gz";
+      md5 = "3da72e2fe16781fe5c7b3b0c6c40ee7b";
+    };
+
+    meta = {
+      description = "Easy to use python subprocess interface";
+      homepage = "https://github.com/ponty/EasyProcess";
```

Comment:
As a matter of fact, nearly all the urls are quoted in this file. The only unquoted ones do not contain antiquotations. Should I really remove these ?